packages/prisma/schema.prisma

```diff
@@ -431,6 +431,8 @@ model Booking {
   /// @zod.custom(imports.bookingMetadataSchema)
   metadata              Json?
   isRecorded            Boolean              @default(false)
+  iCalUID               String?              @default("")
```

Comment:
nit: Why are we giving it a default of empty string? It is marked to accept NULL values already by having ?.

Semantically, it makes more sense to not set the value or keep it NULL(because an empty iCalUID isn't a valid iCalUID anyway). AFAIU, a booking should have iCalUID set going forward always.

Reply:
I also agree that with the way this PR is setup all bookings will have a iCalUID. There were some instances where if we were rescheduling or working with bookings without an iCalUID, it would throw a Prisma error.